src/commentsAPI.js

```diff
@@ -0,0 +1,26 @@
+export default class commentsApi {
+  static fetchRecipes = async (item, appID) => {
+    const res = () => fetch(
+      `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${appID}/comments?item_id=${item}`,
+    )
+      .then((res) => res.json())
+      .then((json) => json);
```

Comment:
- [ ] <b>REQUIRED:</b> As requested in the previous review, you have to use `async/await` in case of any API request in this project. Kindly do that to comply with the project specifications.